EIPS/eip-1884.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 1884
+title: Opcode repricing for trie-size-dependent opcodes 
+author: Martin Holst Swende (@holiman)
+type: Standards Track
+category: Core
+discussions-to: https://ethereum-magicians.org/t/opcode-repricing/3024
+status: Draft
+created: 2019-03-28
+---
+
+
+## Simple Summary
+
+This EIP proposes repricing certain opcodes, to obtain a good balance between gas expenditure and resource consumption.
+
+## Abstract
+
+The growth of the Ethereum state has caused certain opcodes to be more resource-intensive at this point than 
+they were previously. This EIP proposes to raise the `gasCost` for those opcodes.
+
+## Motivation
+
+An imbalance between the price of an operation and the resource consumption (CPU time, memory etc)
+has several drawbacks:
+
+- It could be used for attacks, by filling blocks with underpriced operations which causes excessive block processing time.
+- Underpriced opcodes cause a skewed block gas limit, where sometimes blocks finish quickly but other blocks with similar gas use finish slowly.
+
+If operations are well-balanced, we can maximise the block gaslimit and have a more stable processing time.
+
+## Specification
+
+### Version A
+
+At block `N`, 
+
+- The `SLOAD` operation changes from `200` to `800` gas,
+- The `BALANCE` operation changes from `400` to `700` gas.
+
+
+### Version B
```

Comment:
It's difficult to discuss an EIP that describes two different solutions. Instead of listing version B, perhaps it's a good idea to list version A as EIP-1884 and separate the `SELFBALANCE` as a separate EIP that can be included in addition to EIP-1884.

Reply:
I'm removing section `A`, and going only with B instead